### PR TITLE
feat: #1345 add userName, wayName validate

### PR DESCRIPTION
**Pull request by AKiroi**
- add validate for length userName, wayName
- create getValidValue
- refactor Title, EditableText, EditableTexterea, Label
- add validator

### CODE REVIEW

**Comment by tritonJS826** [comment]
Required another approach -separate validators

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/editableText/EditableText.tsx [hunk 1]
```diff
@@ -121,24 +117,14 @@ export const EditableText = <T extends string | number>(props: EditableTextProps
...
-    isInvalidTextLength
-      ? displayNotification({
-        text: notificationText,
-        type: NotificationType.INFO,
-      })
-      : setValue(updatedValue);
+    if (!props.validators) {
+      setValue(updatedValue);
+    } else {
+      if (typeof updatedValue === "string") {
```
Comment (tritonJS826) [inline mw-webapp/src/component/editableText/EditableText.tsx:123]: we should validate input number too

File: mw-webapp/src/component/editableText/EditableText.tsx [hunk 2]
```diff
@@ -121,24 +117,14 @@ export const EditableText = <T extends string | number>(props: EditableTextProps
...
+        const error = validateValue({value: updatedValue, validators: props.validators});
+        error ? displayValidationError?.(error) : setValue(updatedValue);
+      }
+    }
```
Comment (tritonJS826) [inline mw-webapp/src/component/editableText/EditableText.tsx:127]: we can create separate updateValueWithValidatorsHamdler and reuse it in different componsnts

Comment (tritonJS826) [reply mw-webapp/src/component/editableText/EditableText.tsx:127]: you cant use displayValidationError inside (and get rid of displayValidationError itself and use just it body)

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/editableText/EditableText.tsx (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/src/component/editableText/EditableText.tsx]: done

File: mw-webapp/src/component/editableText/EditableText.tsx (hunk 2 above)
Comment (AKiroi) [reply mw-webapp/src/component/editableText/EditableText.tsx]: done

File: mw-webapp/src/utils/validatorsValue/validators.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    const numValue = typeof value === "string" ? parseFloat(value) : value;
+
+    return !isNaN(numValue) && numValue < minValue ? errorMessage : null;
+  };
+};
+
+/**
+ * Validator checks if a value has a maximum value.
+ */
+export const manValueValidator = (maxValue: number, errorMessage: string): ValidatorValue => {
```
Comment (Ekaterina1994) [inline mw-webapp/src/utils/validatorsValue/validators.ts:39]:
```suggestion
export const maxValueValidator = (maxValue: number, errorMessage: string): ValidatorValue => {
```

**Review by Ekaterina1994 — APPROVED** [review]

File: mw-webapp/src/utils/validatorsValue/validators.ts (hunk 3 above)
Comment (AKiroi) [reply mw-webapp/src/utils/validatorsValue/validators.ts]: done

**Pull request merged**